config/samples/perf_v1alpha1_ocplogtest.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: perf.kubestone.xridge.io/v1alpha1
+kind: OcpLogtest
+metadata:
+  name: ocplogtest-sample
+spec:
+  image:
+    name: quay.io/mffiedler/ocp-logtest:latest
```

Comment:
It might be a better approach to have an exact version referred here for compatibility reasons:
 if they change the call signature of the script and updating latest we will break.

Reply:
Agreed